sitemap-dynamic.ts

```diff
@@ -46,13 +46,17 @@ export default defineNuxtModule({
     nuxt.hook('nitro:build:before', (nitro) => {
       const paths = []
       nitro.hooks.hook('prerender:route', (route) => {
-        if (!route.route.includes('/api/_content')) {
+        // Exclude paths which contain /api/_content, _payload.js and the standard 200.html entry point
+        // None of these should be in the sitemap.xml file
+        if (!route.route.includes('/api/_content') && !route.route.includes('_payload.js') && !route.route.includes('200.html')) {
           paths.push({ path: route.route })
         }
       })
```

Comment:
Good catch 👍  I haven't noticed those `_payload.js` and `200.html` in generated sitemap.

What about gathering all those excluded keywords in an array for better readability ?

```
      const paths = []
      const EXCLUDED_KEYWORDS = ['/api/_content', '_payload.js', '200.html']
      nitro.hooks.hook('prerender:route', (route) => {
        const shouldBeAddedToSitemap = EXCLUDED_KEYWORDS.every(
          (excudedKeyword) => !route.route.includes(excudedKeyword)
        )
        if (shouldBeAddedToSitemap) {
          paths.push({ path: route.route })
        }
      })
```